traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -100,6 +103,22 @@ type ConfigTrafficOpsGolang struct {
 	CRConfigEmulateOldPath bool `json:"crconfig_emulate_old_path"`
 }
 
+// ConfigTO contains information to identify Traffic Ops in a network sense.
+type ConfigTO struct {
+	BaseURL               rfc.URL          `json:"base_url"`
```

Comment:
Should we be using consistent types for our URL representations? in Config we use *url.URL for URL. Unsure of the difference

Reply:
`rfc.URL` is just a wrapper around `url.URL` that provides methods for encoding and decoding as JSON. You can, in fact, cast it to `url.URL`.

Reply:
ahh duh - sorry about that :) need to look closer :)